src/app/services/server-api.service.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { TestBed, inject } from '@angular/core/testing';
+
+import { ServerApiService } from './server-api.service';
+
+describe('AppointmentsService', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [ServerApiService]
+    });
+  });
+
+  it('should be created', inject([ServerApiService], (service: ServerApiService) => {
+    expect(service).toBeTruthy();
+  }));
```

Comment:
Can you add an actual test case showing that the class works?  Eg. calling the Hello World API. Ofc we'll need to change it later once we have an actual API to call.

Reply:
Sure thing.

Reply:
And should this be a mocked test? Or a function within server-api.service.ts?

Reply:
I dunno how the existing tests work, I just want the code to actually be run somehow, and to be able to repeat that run. Code that has never been run is strictly theoretical code.

Reply:
I've added the Appointment service that the code effectively runs. I plan to add tests. However, I think test functions are something we can add in during cleanup week.